examples/dag/dag.yaml

```diff
@@ -7,10 +7,6 @@ connections:
     api_key: ${oc.env:ANTHROPIC_API_KEY}
 
 prompts:
-  openai-ai-prompt:
```

Comment:
keep both examples as we support both

Reply:
I left other 3 prompts in `prompts` section, but this one removed as it used as inline prompt